lti_emailer/settings/base.py

```diff
@@ -307,3 +307,5 @@
 
 CACHE_KEY_MESSAGE_HANDLED_BY_MESSAGE_ID_AND_RECIPIENT = "lti_emailer:message-handled:%s:%s"
 CACHE_KEY_MESSAGE_HANDLED_TIMEOUT = 60 * 60 * 8  # 8 hours
+
+NO_REPLY_ADDRESS = SECURE_SETTINGS.get('no_reply_address', 'no-reply@coursemail.harvard.edu')
```

Comment:
Can't we just construct this based on `LISTSERV_DOMAIN`?  Not a blocker to get a hotfix pushed out, but not my preferred fix.